faq.md

```diff
@@ -5,64 +5,150 @@ layout: faq
 1. TOC
 {:toc}
 
-# What is this website?
+# Overview Questions
 
-This website is a collection of tutorials developed and maintained by the worldwide Galaxy community. You can learn more about this effort with reading our [article](https://www.biorxiv.org/content/early/2018/04/05/225680).
+## What is this website?
 
-# How can I get some help?
+This website is a collection of tutorials developed and maintained by the [worldwide Galaxy community](https://galaxyproject.org/). You can learn more about this effort by reading our [article](https://www.biorxiv.org/content/early/2018/04/05/225680).
 
-If you have any questions, you can reach us using the [Gitter chat](https://gitter.im/Galaxy-Training-Network/Lobby?utm_source=share-link&utm_medium=link&utm_campaign=share-link).
+## What is Galaxy?
 
-# How can I use these ressources to learn about Galaxy and data analyses?
+[Galaxy](https://galaxyproject.org/) is an open data integration and analysis platform for the life sciences, and it is particularly well-suited for data analysis training in life science research.
 
-- Parse the website to find interesting tutorials
-- Find a Galaxy instance to use for the tutorial 
-- Take one of our interactive tour directly on a Galaxy instance
+## What are the tutorials for?
 
-# How can I use these ressources as an instructor?
+These tutorials can be used for learning and teaching how to use Galaxy for general data analysis, and for learning/teaching specific domains such as assembly and differential gene expression analysis with RNA-Seq data.
+
+## What audiences are the tutorials for?
+
+There are two distinct audiences for these materials.
+
+1. **Self-paced individual learners.** These tutorials provide everything you need to learn a topic, from explanations of concepts to detailed hands-on exercises.
+2. **Instructors.** They are also designed to be used by instructors in teaching/training settings.  Slides, and detailed tutorials are provided.  Some tutorials also include computational support such as Docker images that can be used to scale the lessons up to many participants.
```

Comment:
By changing by: 
"Most tutorials also include computational support with the needed tools, data as well as Docker images that can be used to scale the..."